### ISSUE TITLE
Flux

### ISSUE THREAD

**Issue by lhjlhj11** [bmaltais/kohya_ss#2913]
Can this used for flux lora training

**Comment by uzitotte** [comment]
Yes but it's unclear how. I dont get it as an alternative on Linux when chosing the model.

Do I need to apply a config or something?

**Comment by b-fission** [comment]
> Yes but it's unclear how. I dont get it as an alternative on Linux when chosing the model.
> 
> Do I need to apply a config or something?

You need to be on the sd3-flux.1 branch to train with Flux. 
Switch over to it by doing `git checkout sd3-flux.1`
